shiny/ui/dataframe/js/build.ts

```diff
@@ -0,0 +1,27 @@
+import { build } from "esbuild";
+import { sassPlugin } from "esbuild-sass-plugin";
+import * as fs from "node:fs";
+
+async function bundle() {
+  try {
+    const options = {
```

Comment:
I think we need to add `format: "esm"`. esbuild's default is `iife` (which seems to work OK, though)

Reply:
OK, I can try that.